docs/src/development/variables/_index.md

```diff
@@ -178,18 +178,24 @@ such as enabling errors and configuring the Xdebug extension.
 
 To apply a setting to all environments or have a setting differ among multiple PHP containers in one project,
 specify the variables in the `.platform.app.yaml` file for your application.
-See the [PHP configuration page](../../languages/php/ini.md) for more information.
+For more information, see how to [customize PHP settings](../../languages/php/_index.md#customize-the-php-settings).
 
 ### Framework-specific variables
 
-For specific frameworks, you can implement logic to override global configurations with environment-specific variables.
+For specific frameworks, you can implement logic to override global configurations with environment-specific [variables](../../development/variables/set-variables.md#create-environment-specific-variables).
+So you can use the same codebase and settings for all your environments,
+but still adapt the behavior to each environment.
 
-The Drupal templates show examples of logic mapping variables to Drupal's configuration system.
+The Drupal templates show examples of overriding variables from Drupal's configuration system.
 You can apply similar logic for other frameworks.
 
-For [Drupal 8+](https://github.com/platformsh-templates/drupal8/blob/master/web/sites/default/settings.platformsh.php),
-any variable that begins with `drupal:` or `d8settings:` is mapped to the global `$settings` array, for low-level configuration.
-Any variable that begins with `d8config:` is mapped to the global `$config` array,
-which is useful for overriding Drupal's exportable configuration system.
-In this case, the variable name needs two colons, one for `d8config:` and one for the name of the configuration object to override.
-For example, a variable named `d8config:system.site:name` overrides the `name` property of the `system.site` configuration object.
+For Drupal 9 variable override:
+
+- The variable needs three colons: one for the prefix, one for the setting to override and one for the property.
```

Comment:
:lady_beetle: If the prefix is added using a CLI flag, these instructions might be slightly misleading.